src/store/metadata/stub/content/editions/child-obesity_2023-11.json

```diff
@@ -0,0 +1,122 @@
+      "@type": "dcat:Dataset",
+      "in_series": "https://staging.idpd.uk/datasets/children-prevalence-of-weight-status",
+      "identifier": "2023-11",
+      "title": "Children prevalence of weight status",
+      "summary": "Proportion of children aged 4-5 years (reception age) and 10-11 years (Year 6 age) classified as underweight, healthy weight, overweight or obese (including severe obesity), England, academic years 2006/07 to 2022/23.",
+      "description": "This dataset shows the proportion of children aged 4-5 years (reception age) and 10-11 years (Year 6 age) classified as underweight, healthy weight, overweight or obese (including severe obesity), in England, for academic years 2006/07 to 2022/23.",
+      "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+      "creator": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+      "contact_point": {
+        "name": "Office for health improvement and disparities",
+        "email": "mailto:dhsc.publicenquiries@dhsc.gov.uk"
+      },
+      "topics": [
+        "https://staging.idpd.uk/topics/health-and-social-care"
+      ],
+      "frequency": "annual",
+      "keywords": [
+        "obesity",
+        "schools",
+        "national-programme",
+        "child-measurement",
+        "state-funded",
+        "subnational",
+        "local-authority",
+        "county",
+        "region"
+      ],
+      "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+      "issued": "2023-11-06T11:20:33Z",
+      "modified": "2023-11-23T11:00:00Z",
+      "spatial_resolution": ["N09","S12", "W06", "E92", "K02"],
+      "spatial_coverage": "K02000001",
+      "temporal_resolution": ["P1Y"],
+      "temporal_coverage": {
+        "start": "2006-01-01T09:00:00Z",
+        "end": "2022-12-31T12:00:00Z"
+      },
+      "versions_url": "https://staging.idpd.uk/datasets/children-prevalence-of-weight-status/editions/2023-11/versions",
+      "versions": [
+        {
+          "@id": "https://staging.idpd.uk/datasets/children-prevalence-of-weight-status/editions/2023-11/versions/1",
+          "issued": "2023-11-06T11:20:33Z"
+        }
+      ],
+      "next_release": "2024-11-01T09:30:00Z",
+      "table_schema": {
+        "columns": [
+          {
+            "name": "period",
+            "datatype": "string",
+            "titles": "Gregorian Time Interval",
+            "description": "The time period associated with the observation."
+          },
+          {
+            "name": "areacd",
+            "datatype": "string",
+            "titles": "ONS Geography Code",
+            "description": "The statistical geography associated with the observation."
+          },
+          {
+            "name": "areanm",
+            "datatype": "string",
+            "titles": "Area Name",
+            "description": "The name associated with The geography code "
```

Comment:
Trailing whitespace at the end of the description.

Reply:
Done